doc/api/crypto.md

```diff
@@ -1847,7 +1847,7 @@ added: v7.10.0
 changes:
   - version: v9.0.0
     pr-url: https://github.com/nodejs/node/pull/15231
-    description: The `buffer` argument may be any ArrayBufferView
+    description: The `buffer` argument may be any `ArrayBufferView`
 -->
 
 * `buffer` {Buffer|Uint8Array|ArrayBufferView} Must be supplied.
```

Comment:
While at it, `Uint8Array` is now redundant. Ditto below.

Reply:
So is `Buffer`. ;) The thing is, at this point most Node developers don’t know that `Buffer` is a kind of `Uint8Array`, so it makes sense to list them both. In the same vein, keeping `ArrayBufferView` listed explicitly might be redundant but helpful, since 99 % of the time the user *is* going to pass in an `Uint8Array`.

Reply:
But if it is fairly obvious that `Uint8Array` is a type of `TypedArray`. In conjunction with https://github.com/nodejs/node/pull/17595#discussion_r155964729, my preference would be `{Buffer|TypedArray|DataView}`.